src/ui.button-toggle-item/config.ts

```diff
@@ -9,6 +9,7 @@ export default /*tw*/ {
     base: "{UButton} font-normal !bg-{color}-600/15",
     defaults: {
       variant: "thirdary",
+      color: "brand",
```

Comment:
Config sets no color and vite plugin don't know which color to add to safe list, that the reason why anything was added to safelist